### PR TITLE
feat(Slider): Added slider component

### PR DESCRIPTION
**Pull request by HHogg**
slider

### CODE REVIEW

File: packages/axiom-components/src/Slider/Slider.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    case 'ArrowDown':
+      delta = -1;
+      break;
+    case 'ArrowRight':
+    case 'ArrowUp':
+      delta = 1;
+      break;
+    }
+
+    onChange(Math.max(min, Math.min(value + step * delta, max)));
```
Comment (lpoulter) [inline packages/axiom-components/src/Slider/Slider.js]: Out of curiosity why do we need the delta here?

File: packages/axiom-components/src/Slider/Slider.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  user-select: none;
+}
+
+.ax-slider__track {
+  position: absolute;
+  top: 50%;
+  right: 0;
+  left: 0;
+  transform: translateY(-50%);
+  background-color: var(--color-theme-background--subtle);
```
Comment (lpoulter) [inline packages/axiom-components/src/Slider/Slider.css]: I wonder if we want `cursor: pointer` here also as clicking on the track moves the slider.

**Comment by lpoulter** [comment]
Looks great. We have a slider in the `Word cloud` app that has a text input box to allow greater precision. It looks like that could be added at a later date thou.

File: packages/axiom-components/src/Slider/Slider.js (hunk 1 above)
Comment (HHogg) [reply packages/axiom-components/src/Slider/Slider.js]: It sets the direction for when using the arrow keys. It was to shortcut writing `onChange(Math.max(min, Math.min(value +/- step, max)));` twice. Happy to change if the other way is more clearer though.

Comment (lpoulter) [reply packages/axiom-components/src/Slider/Slider.js]: Ok thanks for the explanation. The second way is a little clearer to me.

File: packages/axiom-components/src/Slider/Slider.css (hunk 2 above)
Comment (HHogg) [reply packages/axiom-components/src/Slider/Slider.css]: Good shout. Have added in 👍

**Comment by HHogg** [comment]
Ooops I forgot to add [Netlify link](https://slider--hhogg-axiom.netlify.com/)

**Pull request merged**